src/components/OuterModal/WaitForTxApproval.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useRef, useEffect, useLayoutEffect } from 'react'
+import styled from 'styled-components'
+
+import Modal from 'components/common/Modal'
+import { openGlobalModal, useGlobalModalContext, setOuterModalContext, closeGlobalModal } from './SingletonModal'
+
+const WaitForTxWrapper = styled.div`
+  font-size: 1.3em;
+`
+
+interface GlobaModalContextSlice {
+  pendingTxApprovals: Set<number | string>
+}
+
+const PenidngTxApprovalsMessage: React.FC<{ txNumber: number }> = ({ txNumber }) => {
+  return (
+    <>
+      <h4>No response from wallet for pending transaction</h4>
+      <p>There are currently {txNumber} transactions waiting for approval for your wallet.</p>
+      <p>
+        Mesa has detected a potential transaction timeout. Please validate with your wallet that you have properly
```

Comment:
I would drop this phrase completly. `Mesa has detected a potential transaction timeout`

Also, not nice to hardcode "Mesa", Mesa is dxDAO product not ours